docs/running-calyx/fud2/scripts.md

```diff
@@ -28,6 +28,66 @@ export const verilog_state = state("verilog", ["sv", "v"]);
 
 These two lines define a `calyx` state and a `verilog` state. The `export` prefix means that these variables will be accessible to other scripts that `import "calyx"`.
 
+Now we will define an operation taking the `calyx` state to the `verilog` state. These operations define functions where the signiture asserts the result of running the body.
+
+```rust,ignore
+// define an op called "calyx_to_verilog" taking a "calyx_state" to a "verilog_state".
+defop calyx_to_verilog(c: calyx_state) >> v: verilog_state {
+    // retrieve a variable from the fud2.toml config
+    let calyx_base = config("calyx.base");
+    // retrieve a variable from the config, or a default derived from calyx-base
+    let calyx_exe = config_or("calyx.exe", "${calyx_base}/target/debug/calyx");
+    // retrieve a variable from cli options, or with a default
```

Comment:
To nitpick, _all_ of these could come from either the config file or the command-line options…